hydra-http-form.c

```diff
@@ -509,8 +510,17 @@ int32_t parse_options(char *miscptr, ptr_header_node *ptr_head) {
       hydra_report(stderr, "[ERROR] Out of memory for HTTP headers (H).\n");
       return 0;
     default:
-      hydra_report(stderr, "[ERROR] no valid optional parameter type given: %c\n", miscptr[0]);
-      return 0;
+      while (*ptr != 0 && *ptr != ':')
```

Comment:
if an option is unknown an error should be printed and exited dunno why you changed the behaviour

Reply:
The parameter will actually be validated in both modules. When I set r=200, this parameter is not in hydra-http-form.c, so I have to ignore it. It will be validated in hydra-http.c

Reply:
yes but I can also do x=fooo which is a non-existing parameter and there will be no information about this. also if you use r= in http-form there will not even be a warning/error that this option is not supported there.